src/utils/hashes/index.ts

```diff
@@ -79,76 +113,79 @@ export const computeTransactionHash = (txJSON: any): string => {
  * @param txBlobHex - The binary transaction blob as a hexadecimal string.
  * @returns The hash to sign.
  */
-export const computeBinaryTransactionSigningHash = (
-  txBlobHex: string,
-): string => {
+export function computeBinaryTransactionSigningHash(txBlobHex: string): string {
   const prefix = HashPrefix.TRANSACTION_SIGN.toString(16).toUpperCase()
   return sha512Half(prefix + txBlobHex)
 }
 
 /**
- * Compute Account Root Index.
+ * Compute AccountRoot Ledger Object ID.
```

Comment:
I think `index` is still more correct than `ID`, because that's what rippled calls it.

Reply:
Same for all the other methods here.